test/stream/api/source/rollback-version-404/005-poll-rollback/expected-response.json

```diff
@@ -0,0 +1,11 @@
+{
+  "taskId": "api.rollback.sourceVersion/%%TEST_DEFAULT_BRANCH_ID%%/my-source/%s",
+  "type": "api.rollback.sourceVersion",
+  "url": "https://stream.keboola.local/v1/tasks/api.rollback.sourceVersion/%%TEST_DEFAULT_BRANCH_ID%%/my-source/%s",
+  "status": "error",
+  "isFinished": true,
+  "createdAt": "%s",
+  "finishedAt": "%s",
+  "duration": %d,
+  "error": "Source version \"my-source/0000012345\" not found in the branch."
```

Comment:
The check if version exists should happen immediately and not only in the background task. `004-rollback-version` should return `404`. Enable and Disable on Source and Sink work this way as well.